frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -710,6 +710,38 @@ public void testLoadModelFromURL() throws InterruptedException {
         Assert.assertTrue(new File(configManager.getModelStore(), "squeezenet1_1.mar").exists());
     }
 
+    @Test(
+            alwaysRun = true,
+            dependsOnMethods = {"testModelRegisterWithDefaultWorkers"})
+    public void testLoadModelFromFileURI() throws InterruptedException {
+        String curDir = System.getProperty("user.dir");
+        File curDirFile = new File(curDir);
+        String parent = curDirFile.getParent();
+
+        String source = configManager.getModelStore() + "/mnist.mar";
+        String destination = parent + "/modelarchive/mnist1.mar";
+        File sourceFile = new File(source);
+        File destinationFile = new File(destination);
+        String fileUrl = "";
+        try {
+            FileUtils.copyFile(sourceFile, destinationFile);
+            fileUrl = "file://" + parent + "/modelarchive/mnist1.mar";
+        } catch (Exception e) {
+            Assert.assertEquals(e.getClass(), IOException.class);
+        }
+        testLoadModel(fileUrl, "mnist1", "1.0");
+        Assert.assertTrue(new File(configManager.getModelStore(), "mnist1.mar").exists());
+        FileUtils.deleteQuietly(destinationFile);
```

Comment:
clean up the fileUrl as well.

Reply:
@harshbafna In this testcase, we copied mnist.mar from resources folder as mnist1.mar. In the end we have deleted mnist1.mar. fileUrl variable is just a URL like "file://path) .